ios/PolkadotVault.xcodeproj/project.pbxproj

```diff
@@ -2477,7 +2477,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 15.6;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.5;
```

Comment:
downgrade?

Reply:
Yes, as this would exclude some iPod Touch models, without any upside